### PR TITLE
Merge master into develop

### PR DESCRIPTION
**Pull request by johnwatkins0**
### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jeffpaul** [comment]
Pinging @rickalee and @oscarssanchez to review these as there's a good bit being removed that I want to make sure isn't done mistakenly.  I'm working on reviewing the changes but would feel more confident if someone with history with the plugin could give this a review as well.

File: .jshintrc [hunk 1]
```diff
@@ -1,25 +0,0 @@
-{
```
Comment (jeffpaul) [inline .jshintrc]: We certain it's ok to fully remove this file?

File: package.json [hunk 2]
```diff
@@ -18,18 +18,13 @@
 		"url": "https://github.com/10up/windows-azure-storage"
 	},
 	"homepage": "https://github.com/lukaspawlik/windows-azure-storage#readme",
-	"devDependencies": {
-		"grunt": "^0.4.5",
-		"grunt-contrib-concat": "^0.5.1",
-		"grunt-contrib-uglify": "^0.9.1",
-		"grunt-postcss": "^0.6.0",
-		"grunt-contrib-cssmin": "^0.12.3",
-		"grunt-contrib-jshint": "^0.11.2",
```
Comment (jeffpaul) [inline package.json]: We certain that we no longer require some of these dependencies?

File: tasks/options/clean.js [hunk 3]
```diff
@@ -1,3 +0,0 @@
-module.exports = {
-	main: ['release/<%= pkg.version %>']
-};
```
Comment (jeffpaul) [inline tasks/options/clean.js]: We certain it's ok to fully remove this file?

File: tasks/build.js [hunk 4]
```diff
@@ -1,3 +0,0 @@
-module.exports = function (grunt) {
-	grunt.registerTask( 'build', ['default', 'clean', 'copy', 'compress'] );
-};
```
Comment (jeffpaul) [inline tasks/build.js]: We certain it's ok to fully remove this file?

File: tasks/options/copy.js [hunk 5]
```diff
@@ -1,28 +0,0 @@
...
-			'!composer.json',
-			'!composer.lock',
-			'!Gruntfile.js',
-			'!package.json',
-			'!phpunit.xml',
-			'!phpunit.xml.dist'
-		],
-		dest: 'release/<%= pkg.version %>/'
-	}
-};
```
Comment (jeffpaul) [inline tasks/options/copy.js]: We certain it's ok to fully remove this file?

File: tasks/options/jshint.js [hunk 6]
```diff
@@ -1,6 +0,0 @@
-module.exports = {
-	all: [
-		'Gruntfile.js',
-		'js/src/**/*.js',
-	]
-};
```
Comment (jeffpaul) [inline tasks/options/jshint.js]: We certain it's ok to fully remove this file?

File: tasks/options/compress.js [hunk 7]
```diff
@@ -1,12 +0,0 @@
...
-		options: {
-			mode: 'zip',
-			archive: './release/drweil.<%= pkg.version %>.zip'
-		},
-		expand: true,
-		cwd: 'release/<%= pkg.version %>/',
-		src: ['**/*'],
-		dest: 'drweil/'
-	}
-};
```
Comment (jeffpaul) [inline tasks/options/compress.js]: We certain it's ok to fully remove this file?

File: readme.txt [hunk 8]
```diff
@@ -1,19 +1,17 @@
 === Microsoft Azure Storage for WordPress ===
-Contributors: msopentech, 10up, morganestes, stevegrunwell, lpawlik, rittesh.patel
```
Comment (jeffpaul) [inline readme.txt]: I prefer to maintain previously-listed contributors unless they've asked to be removed.  Any reason we should NOT add these folks back (as well as any others newly significantly contributing)?

File: tasks/options/compress.js (hunk 7 above)
Comment (rickalee) [reply tasks/options/compress.js]: Confirmed build process updates work. Pull request #101 added to further update dependencies.

**Comment by jeffpaul** [comment]
@johnwatkins0 will likely need to update here once #101 gets merged

**Comment by rickalee** [comment]
Spot checks on master looks good and diff looks to include all the recent changes around multisite, special characters in file names, and srcset fixes. ✅

**Pull request merged**